<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialDock</class>
 <widget class="QDockWidget" name="MaterialDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="LightColorWidget" name="colorWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="shininessGroupBox">
      <property name="title">
       <string>Shininess</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="shininessLayout">
         <item>
          <widget class="QSlider" name="shininessSlider">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>1280</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="shininessDoubleSpinBox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>128.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightColorWidget</class>
   <extends>QWidget</extends>
   <header>LightColorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
